request/util.js

```diff
@@ -10,13 +10,20 @@ define([
 	'../has'
 ], function(exports, RequestError, CancelError, Deferred, ioQuery, array, lang, Promise, has){
 
+	var HAS_FORM_ELEMENT = !!window.HTMLFormElement;
```

Comment:
Sorry that I missed this before, but I think this would fail in the Node.js content, as window is not a valid global.